### ISSUE TITLE
Build failures for environments

### ISSUE THREAD

**Issue by marionebl** [sinnerschrader/patternplate-server#17]
build for environments fails with 

```
error: [patternplate] TypeError: Cannot read property 'manifest' of undefined
    at _loop (/node_modules/patternplate/node_modules/patternplate-server/application/tasks/build/index.js:422:23)
```

This appears to be an error shadowed by `getPatterns` failing silently for builds of virtual environment patterns. Make sure getPatterns fails explicitly and investigate further

**Comment by timbru31** [comment]
@pepe79 FYI

**Comment by marionebl** [comment]
This is addressed in #18
